main.nf

```diff
@@ -164,6 +165,11 @@ workflow {
     )
 }
 
+output {
+    directory params.outdir
+    mode params.publish_dir_mode
+}
```

Comment:
Can this be set in `nextflow.config` @bentsherman?

Reply:
Not currently, but should be considered as a future improvement. If we don't need the target-specific config in the output block, would be better IMO to make these config settings instead of pipeline code + params.

Reply:
I agree that they should be config. Much less flexible for the user if you have to define a param just to make it configurable

For now you can customize things like the mode for specific targets, though that it not as granular as what you describe

Reply:
These settings have been moved to the config